image/cli/mascli/functions/pipeline_config_applications

```diff
@@ -137,6 +137,10 @@ function config_pipeline_applications() {
 
   # Manage
   if prompt_for_confirm "Install Manage"; then
+    echo "Maximo IT License Terms"
```

Comment:
Please move this logic into the step where we configure Manage components.  We can use the components the use selected to determine if Maximo IT is being installed (it's just a component in Manage right?) This will allow us to show this only when it's meaningful to the user (they have added the Maximo IT component to the install).